public/components/toolbar/spatial_filter/filter_by_polygon.tsx

```diff
@@ -12,22 +12,29 @@ import {
   DRAW_FILTER_POLYGON,
   DRAW_FILTER_POLYGON_DEFAULT_LABEL,
   DRAW_FILTER_SPATIAL_RELATIONS,
-  DRAW_FILTER_SHAPE_TITLE,
+  DRAW_FILTER_SHAPE_TITLE, DRAW_FILTER_CANCEL,
 } from '../../../../common';
 import { FILTER_DRAW_MODE } from '../../../../common';
 
 interface FilterByPolygonProps {
   setDrawFilterProperties: (properties: DrawFilterProperties) => void;
-  isDrawActive: boolean;
+  mode: FILTER_DRAW_MODE;
 }
 
-export const FilterByPolygon = ({
-  setDrawFilterProperties,
-  isDrawActive,
-}: FilterByPolygonProps) => {
+const isFilterActive = (mode: FILTER_DRAW_MODE): boolean => {
+  return mode === FILTER_DRAW_MODE.POLYGON;
+};
```

Comment:
We can directly use a boolean instead of function.
```suggestion
const isFilterActive = mode === FILTER_DRAW_MODE.POLYGON;
```

Reply:
Ack